public/js/varien/js.js

```diff
@@ -835,6 +835,10 @@ class Template
      * @returns {string} Interpolated string
      */
     evaluate(data = {}) {
+        // if (typeof Hash !== 'undefined' && data instanceof Hash) {
```

Comment:
```suggestion
```

Reply:
sorry, I read that it's there on purpose

Reply:
Oh yeah, I'll totally remove it though. I just wanted to point out it was my initial solution, but I think checking for a `data?.toObject()` method (which prototypejs has), is the better solution right?